content/examples/cards.mdx

```diff
@@ -200,29 +200,29 @@ lastUpdatedDate: 2023-05-25T12:00:00.000Z
   </Section>
   <Section heading="Library Card" width="full">
   ```html
-  <div class="card block w-full basefont-23 leading-display bg-white text-black border border-solid border-black-10 shadow-md md:max-w-500">
-    <div class="overflow-hidden aspect-[16/9] relative" aria-hidden="true">
+  <div className="card block w-full basefont-23 leading-display bg-white text-black border border-solid border-black-10 shadow-md md:max-w-500">
```

Comment:
@jdwjdwjdw 
It looks like you've changed the example markup from HTML valid markup to React markup. Can you tell me more about why you changed these? This changes the example markup code and they are invalid HTML attributes. 

![Screenshot 2024-05-29 at 09 43 02](https://github.com/SU-SWS/decanter-web/assets/550602/de3475b9-7d68-4dc9-9fcc-b96671d4566e)

Reply:
The stuff inside the html fencing `\`\`\`html` should stay as html and not react.

Reply:
Thanks - I got a little overambitious while trying to fix the console errors. Some of the JSX vs HTML did need a little cleanup, so I removed my incorrect changes and kept the ones that needed updating. Should be good for another look now.